ctlog/ctlog.go

```diff
@@ -26,28 +26,52 @@ import (
 	ct "github.com/google/certificate-transparency-go"
 )
 
+// Interval represents the interval [Start, End).
+type Interval struct {
+	Start, End time.Time
+}
+
 // Log contains metadata about a CT Log that is needed by Monologue.
 type Log struct {
 	Name      string
 	URL       string
 	PublicKey crypto.PublicKey
 	MMD       time.Duration
+
+	// TemporalInterval represents the interval in which a certificate's
+	// NotAfter field must fall to be accepted by the Log (as specified by the
+	// Log Operators).
+	// TemporalInterval.Start and TemporalInterval.End are both to second
+	// precision.  Any smaller units may be ignored/discarded.
```

Comment:
By the way, what was the reason for truncating to second precision? At the time of review, I thought it was because that was the precision of certificate NotBefore and NotAfter dates, but I've just learnt that they can have fractional seconds.

Reply:
Log Operators specify their temporal ranges to second precision, and they are reflected that way in the various Log lists.  I figured it was good to be consistent :)